### PR TITLE
Added FirewallD support to scripts

### PR DESCRIPTION
**Pull request by freysteinn**
The default action on Fedora and RHEL-based distributions that use FirewallD is to ban DHCP requests. Instead of telling people to turn off their firewall, I recommend adding the tap interfaces to the FirewallD trusted zone.

This commit adds automatic support to the create_net.sh and cleanup-net.sh scripts that set up the tap interfaces. Due to many distributions using FirewallD these days, I opted for using the following command to check if FirewallD is available:

```bash
if [ -e $(which firewall-cmd) ]; then
    sudo firewall-cmd --zone=trusted --change-interface=$device
fi
```

However, we will have to create a better solution for this in the future.

### CODE REVIEW

**Comment by freysteinn** [comment]
This fixes #387.

**Comment by freysteinn** [comment]
The cleanup-net.sh script uses a different device variable name than the create_net.sh script. Cleanup uses $i, but create uses $device. To prevent us from having to do the pull-request walk of shame in case it goes past us, I decided to make both scripts follow the same convention for the future.

TL;DR
cleanup-net.sh uses $i to refer to the devices while create_net.sh uses $device. This commit updates cleanup-net.sh to use $device to make it easier for the future.

**Comment by freysteinn** [comment]
This should fix #241 as well.

**Comment by freysteinn** [comment]
Apart from waiting, is there anything else I need to do?

**Review by dbaluta — APPROVED** [review]
